libraries/lib-snapping/ProjectSnap.cpp

```diff
@@ -0,0 +1,120 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
```

Comment:
The commit adding this file has an easily resolved conflict in libraries/CMakeLists.txt